CONTRIBUTING.md

```diff
@@ -0,0 +1,119 @@
+# Contribute to Apache Pulsar Site
+
+## How-tos
+
+### How to fix search index mismatches?
+
+First of all, you should get the permission to access `apache_pulsar` crawler on [Algolia Crawler console](https://crawler.algolia.com/). You can email dev@pulsar.apache.org to ask for permission.
+
+The most common fix for search index mismatches is to re-index the pages. You can do so by clicking "Restart crawling" button on the [crawler page](https://crawler.algolia.com/admin/crawlers/7a3458ba-2373-47d5-9520-90cc9cc10736/overview). Typically, it takes about 1 or 2 hours to complete.
+
+### How to update reference pages?
+
+The source of reference pages consists of:
+
+* Generated from configuration classes
+* Generated from command-line interfaces
+* Manually held under `static/referces` folder
+
+Read [update reference docs](https://pulsar.apache.org/contribute/document-contribution/#update-reference-docs) guide for details.
```

Comment:
Suggest consolidating this part into [update reference docs - guide](https://pulsar.apache.org/contribute/document-contribution/#update-reference-docs) to keep a single source of truth.

Reply:
This section is covered by the "Update reference docs" guide. I add this CONTRIBUTING runbook for FAQs/How-tos and links to the related pages.